app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[edit update destroy]
+
+  # GET /users
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1/edit
+  def edit; end
+
+  # PATCH/PUT /users/1
+  def update
+    respond_to do |format|
+      if @user.update(user_params)
+        format.html { redirect_to users_url, notice: "User was successfully updated." }
```

Comment:
If you expects only html requests, then there's no need to use `respond_to`
Let's get rid of it

Reply:
Keep this in mind all around your application 👆🏻

Reply:
Gotcha. Already removed these in this controller. Will remove the same stuff in other controllers in their feature branch.